app/furniture/marketplace/cart/delivery_areas/show.html.erb

```diff
@@ -0,0 +1,2 @@
+<%- breadcrumb :marketplace_cart_delivery_area, cart %>
+<%= render Marketplace::Cart::DeliveryAreaComponent.new(cart: cart) %>
```

Comment:
How would one land one this `show` page? I can't find any links to a standalone show page in this PR.

Reply:
When Turbo fails (or a Shopper disables it... or they command + click/open in a new tab), submitting `cart/delivery_areas#update` redirects them to the `cart/delivery_areas#show` endpoint.

Reply:
Thanks for clarifying!